Add deprecation message (#6)
oscar-system/changelog-script: Readme.md

Action to update the release notes file in oscar-system universe

> [!WARNING]

> > This action will no longer be supported, please use <https://github.com/oscar-system/update-release-notes/> instead.

Reviewer: ```suggestion
> This action is no longer supported, please use <https://github.com/oscar-system/update-release-notes/> instead.
```
